setup.py

```diff
@@ -295,7 +295,12 @@ def libadcc_extension():
             found_libtensor = True
             extra_compile_args.extend(cflags)
             extra_link_args.extend(libs)
-            runtime_library_dirs.extend(extract_library_dirs(libs))
+            if sys.platform == "darwin":
+                extra_link_args.append("-Wl,-rpath,@loader_path")
+                for path in extract_library_dirs(libs):
+                    extra_link_args.append(f"-Wl,-rpath,{path}")
+            else:
+                runtime_library_dirs.extend(extract_library_dirs(libs))
```

Comment:
Classic.... 🙃

Reply:
Indeed, some tricks you know after a while. Let's hope that does it.